### ISSUE TITLE
Pojavlauncher couldnt be installed

### ISSUE THREAD

**Issue by Montivillon** [PojavLauncherTeam/PojavLauncher_iOS#662]
### Describe the bug

Failed to verify code signature of /var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.MUJMYv/extracted/App.app/Frameworks/UnzipKit.framework : 0xe8008015 (A valid provisioning profile for this executable was not found.)

### The log file and images/videos

Idk

### Steps To Reproduce

```markdown
Ikd
```

### Expected Behavior

It cannot be sideloaded with sidestore or altstor

### Platform

```markdown
- Device model: iPad 9th generation 
- iOS version: 17.4
- PojavLauncher version: 3.0
```

### Anything else?

Basically, I couldn’t use altstore to even sideload, it has this error message: Failed to verify code signature of /var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.MUJMYv/extracted/App.app/Frameworks/UnzipKit.framework : 0xe8008015 (A valid provisioning profile for this executable was not found.) I need help to fix

**Comment by Kiokingz** [comment]
please download the latest actions build, your issue has been fixed.
